src/utils/ipfs.ts

```diff
@@ -1,6 +1,7 @@
 class IpfsUploadError extends Error {
```

Comment:
with the addition of the new pinata file, is this one still used ? 
maybe it should be deleted or we need other code updates from ipfs to pinata file

Reply:
I kept the old config for screens and the new one for url uploads.